.gitignore

```diff
@@ -0,0 +1,28 @@
+training.phtml
+/Controllers/FrontControllerVlad.php
+/Controllers/ProjectControllers/BaseControllerYarik.php
+/Controllers/ProjectControllers/IndexControllerYarik.php
+/Controllers/ProjectControllers/UserController1.php
+/Controllers/ProjectControllers/UserControllerVlad.php
+/Controllers/ProjectControllers/UserControllerVlad2.php
+/Controllers/ProjectControllers/UserControllerYarik.php
+/Controllers/ProjectControllers/Admin/AdminController.php
+/Controllers/ProjectControllers/Admin/AdminControllerYarik.php
+/Controllers/ProjectControllers/Admin/IndexControllerYarik.php
+/logs/pdo_error.log
```

Comment:
Are you sure you have to add certain files in the logs directory to a gitignore rather than add whole logs directory?